department-of-veterans-affairs/caseflow
# Ki/APPEALS-29343 - Create Submit Confirmation Modal

Resolves [Create Submit Confirmation Modal](https://jira.devops.va.gov/browse/APPEALS-29434)

# Description
Created Submit Confirmation Modal for Correspondence Intake Form.

## Acceptance Criteria

## Testing Plan
1. Navigate to third step of Intake Form
2. Click on "Submit" button at bottom.
3. Close (with either X or cancel button) and verify modal closes and you "return(stay)" on the third step of the process.
4. Click on Submit on Intake Form page again.
5. Click on Submit on Modal.
    1. Currently does not actually submit data/updates
    2. Redirects back to /queue/correspondence 


# Frontend
## User Facing Changes
![Screenshot 2023-12-05 at 10 17 32 AM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/a99c4b17-2212-4b65-b65e-081fc551bf08)
![Screenshot 2023-12-05 at 10 22 50 AM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/07f1cf4b-c7b8-4bb3-837e-9b79909b8428)
![Screenshot 2023-12-05 at 10 22 34 AM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/df6e653e-72f5-4359-adaf-602e21d1728c)
